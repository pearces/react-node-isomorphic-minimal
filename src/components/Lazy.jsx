import React from 'react';

const Lazy = () => (
  <div>
    <p>This is a lazy loaded component.</p>
  </div>
);

export default Lazy;
