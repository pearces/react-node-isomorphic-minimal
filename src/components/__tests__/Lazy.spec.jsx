import React from 'react';
import renderer from 'react-test-renderer';
import Lazy from '../Lazy';

describe('Lazy', () => {
  it('matches component snapshot', () => {
    const html = renderer.create(<Lazy />);
    const tree = html.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
