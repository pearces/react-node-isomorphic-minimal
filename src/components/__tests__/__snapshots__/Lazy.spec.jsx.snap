// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Lazy matches component snapshot 1`] = `
<div>
  <p>
    This is a lazy loaded component.
  </p>
</div>
`;
